coralproject/talk
# Like button

## What does this PR do?

Adds a like button and the ability to remove a flag to talk.

## How do I test this PR?

1. Like a comment in the stream.
2. The button should change color and the like count should increase.
3. Reload the page. The button should be black but the count should remain.
4. Like and then unlike the comment a second time. The count should decrease to its original number.
5. Reload the page, the appropriate number should display.
6. Flag and unflag a comment. The button should return to its "unflaged" state and you should receive an appropriate notification. 

@coralproject/tech
